lib/tasks/locations.rake

```diff
@@ -0,0 +1,53 @@
+namespace :goodcity do
+
+  # rails goodcity:remove_unused_locations
+  desc 'Delete locations which does not have any associated data'
+  task remove_unused_locations: :environment do
+    Location.find_in_batches(batch_size: 100).each do |locations|
```

Comment:
should we move the task to a new / existing lib file
and support the operations with specs?

Reply:
I think we almost always want to enrich the capabilities of goodcity with tested operations, and call them from rakes